cygnus-twitter/src/main/java/com/telefonica/iot/cygnus/sinks/TwitterBatch.java

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
It seems this is only a wrapper for `ArrayList`. Thus, I'm wondering if you can directly use such `ArrayList`. This would avoid things as seen in `TwitterHDFSSink.persistBatch(Batch)` where the inner array list has to be gotten and then the aray list iterated; in that case, the `persistBatch` method would receive an `ArrayList` instead of a `Batch`.